cmd/plugins/juju-metadata/toolsmetadata.go

```diff
@@ -61,21 +62,16 @@ use the --clean option.
 Examples:
 
   - generate metadata for "released" tools, looking in the "releases" directory:
```

Comment:
Pretty sure this shouldn't said "releases" now. Can the default value (envtools.ReleasedStream) be substituted in?